src/docfx/localization/LocalizationUtility.cs

```diff
@@ -130,6 +130,12 @@ public static bool TryGetSourceRepository(string remote, string branch, out stri
                 {
                     sourceBranch = contributionBranch;
                 }
+
+                if (sourceBranch != "live")
```

Comment:
if it is designed as always fallback to `master`, then we could remove this `live` hack because live will first attempt to fallback to `live` first then `master`?

Reply:
non-live branch fallback to "master", live always fallback to "live"